### PR TITLE
NMS-12874: Renamed RemotePoller to PerspectivePoller

### PR DESCRIPTION
**Pull request by fooker**
* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-12874

### CODE REVIEW

**Comment by fooker** [comment]
Waiting for #3133 to be merged first...

File: features/perspectivepoller/src/test/java/org/opennms/netmgt/perspectivepoller/PerspectivePollerdIT.java [hunk 1]
```diff
@@ -98,7 +98,7 @@
         "classpath:/META-INF/opennms/applicationContext-commonConfigs.xml",
         "classpath:/META-INF/opennms/applicationContext-dao.xml",
         "classpath:/META-INF/opennms/applicationContext-databasePopulator.xml",
-        "classpath:/META-INF/opennms/applicationContext-testRemotePollerDaemon.xml",
+        "classpath:/META-INF/opennms/applicationContext-testPerpsectivePollerDaemon.xml",
```
Comment (christianpape) [inline features/perspectivepoller/src/test/java/org/opennms/netmgt/perspectivepoller/PerspectivePollerdIT.java:101]: Typo

File: features/perspectivepoller/src/test/java/org/opennms/netmgt/perspectivepoller/PerspectivePollerdIT.java [hunk 2]
```diff
@@ -263,66 +263,66 @@ public void reportResultTest() throws Exception {
...
         // Initial config, ICMP and SNMP bound to single package
-        Assert.assertEquals(8, this.remotePollerd.scheduler.getJobKeys(GroupMatcher.anyGroup()).size());
-        assertThat(findRemotePolledService(this.node1icmp, "RDU").getPkg().getName(), is("foo1"));
-        assertThat(findRemotePolledService(this.node1icmp, "Fulda").getPkg().getName(), is("foo1"));
-        assertThat(findRemotePolledService(this.node2icmp, "RDU").getPkg().getName(), is("foo1"));
-        assertThat(findRemotePolledService(this.node2icmp, "Fulda").getPkg().getName(), is("foo1"));
-        assertThat(findRemotePolledService(this.node1snmp, "RDU").getPkg().getName(), is("foo1"));
-        assertThat(findRemotePolledService(this.node2snmp, "RDU").getPkg().getName(), is("foo1"));
+        Assert.assertEquals(8, this.perspectivePollerd.scheduler.getJobKeys(GroupMatcher.anyGroup()).size());
+        assertThat(findperspectivePolledService(this.node1icmp, "RDU").getPkg().getName(), is("foo1"));
```
Comment (christianpape) [inline features/perspectivepoller/src/test/java/org/opennms/netmgt/perspectivepoller/PerspectivePollerdIT.java:309]: upper case P please, findPerspectivePolledService

**Review by christianpape — CHANGES_REQUESTED** [review]
Just a few comments

**Pull request merged**